### ISSUE TITLE
1 Insufficient cpu m5.large - sorry a question not issue

### ISSUE THREAD

**Issue by weicheng113** [GoogleCloudPlatform/spark-on-k8s-operator#922]
Hi,

Sorry to ask question here. May I request to join the slack channel please?(my email: weicheng112@gmail.com)

I tried to run two spark jobs in AWS m5.large(vCPU:2, Memory: 8 GiB) and I got "0/1 nodes are available: 1 Insufficient cpu". At the moment, the core is set to 1. I was trying to reduce it but 1 is the minimum value.

I am only using driver at the moment, as there is a serviceAccount issue(waiting for spark 3).

I guess my case is rare, but we are currently only running small set of service. So we would like to only keep suffient resource. Thanks.

**Comment by liyinan926** [comment]
You can set the cpu request/limit of the driver pod to a value lower than 1:

```
spec:
  driver:
    coreRequest: 0.5
```

**Comment by weicheng113** [comment]
Thanks @liyinan926 , I used "spec: driver: core". I will try that out.

**Comment by weicheng113** [comment]
@liyinan926 The spec.driver.coreRequest is not working. I inspected related yaml:

1. Core Request is correct in the following yaml.
kubectl describe scheduledsparkapplications myScheduledApp
`
Spec:
  Concurrency Policy:            Forbid
  Failed Run History Limit:      3
  Schedule:                      0 0 * * *
  Successful Run History Limit:  1
  ...
    Driver:
      Core Limit:    1200m
      **Core Request:  0.5**
`
2. Core request is not get applied correctly in pod.
kubectl describe pods/myScheduledApp-1590370829543600455-driver
`
...
                spark-role=driver
                sparkoperator.k8s.io/launched-by-spark-operator=true
                sparkoperator.k8s.io/submission-id=dbb8fd6b-d193-4419-9fc2-28d9fea557d9
                ...
    Limits:
      cpu:     1200m
      memory:  896Mi
    **Requests:
      cpu:     1**
      memory:  896Mi
`

I tried the same to my sparkApplication(none scheduled one). Corerequest does not seem to be applied. I don't have a reopen option on the issue. If I don't see reply for some time, I will create another issue. Thanks for your time.
